tests/integration/test_flight_interface_worker.py

```diff
@@ -16,6 +16,7 @@
 FLIGHT_INTERFACE_TIMEOUT = 10.0  # seconds
 FLIGHT_INTERFACE_BAUD_RATE = 57600  # symbol rate
 FLIGHT_INTERFACE_WORKER_PERIOD = 0.1  # seconds
+QUEUE_MAX_SIZE = 1  # Max items allowed in odometry queue
```

Comment:
Rename or remove, current variable name implies queues in general which is contradicted by comment.